cpp/include/cudf/detail/null_mask.cuh

```diff
@@ -156,8 +155,7 @@ size_type inplace_bitmask_binop(
   host_span<bitmask_type const*> masks,
   host_span<size_type const> masks_begin_bits,
   size_type mask_size_bits,
-  rmm::cuda_stream_view stream,
-  rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource())
```

Comment:
Docs to be updated

Reply:
use `@copybrief`, copy-paste the parameter documentation and remove unused params.